src/modules/organization/components/InternalGroupDetail/InternalGroupUserHighlights.tsx

```diff
@@ -1,9 +1,105 @@
+import { INTERNAL_GROUP_USER_HIGHLIGHTS_BY_INTERNAL_GROUP_QUERY } from '../../queries'
+import { FullPageError } from 'components/FullPageComponents'
+import { FullContentLoader } from 'components/Loading'
+import {
+  InternalGroupUserHighlightNode,
+  InternalGroupUserHighlightsByInternalGroupReturns,
+  InternalGroupUserHighlightsByInternalGroupVariables,
+} from '../../types'
+import { InternalGroupUserHighlightEditForm } from 'modules/organization/components/InternalGroupDetail/components/EditHighlights/InternalGroupUserHighlightEditForm'
+import { useState } from 'react'
+import { IconNotes } from '@tabler/icons'
+
 interface InternalGroupUserHighlightsProps {
   internalGroupId: string
 }
 
 export const InternalGroupUserHighlights: React.FC<
   InternalGroupUserHighlightsProps
 > = ({ internalGroupId }) => {
-  return <span>Her kommer beskrivelser om noen folk i gjengen</span>
+  const { data, loading, error } = useQuery<
+    InternalGroupUserHighlightsByInternalGroupReturns,
+    InternalGroupUserHighlightsByInternalGroupVariables
+  >(INTERNAL_GROUP_USER_HIGHLIGHTS_BY_INTERNAL_GROUP_QUERY, {
+    variables: { internalGroupId: internalGroupId },
+  })
+  const [modalOpened, setModalOpened] = useState(false)
+  const [selectedHighlight, setSelectedHighlight] =
+    useState<InternalGroupUserHighlightNode | null>(null)
+  const theme = useMantineTheme()
+  if (error || !data) return <FullPageError />
+  if (loading) return <FullContentLoader />
+
+  const { internalGroupUserHighlightsByInternalGroup: highlightData } = data
+  return (
+    <SimpleGrid
+      cols={3}
+      p={'xl'}
+      breakpoints={[
+        { maxWidth: 900, cols: 1, spacing: 'sm' },
+        { maxWidth: 1200, cols: 2, spacing: 'sm' },
+      ]}
+    >
+      {highlightData.map(highlight => (
+        <Card key={highlight.id} withBorder radius={'lg'}>
+          <Card.Section>
+            {highlight.image && (
+              <Image src={highlight.image.toString()} height={300} />
+            )}
+          </Card.Section>
+          <Group position={'apart'} mt="md" mb="xs">
+            <Text weight={700}>{highlight.user.firstName}</Text>
+            <Group>
+              <Badge>{highlight.occupation}</Badge>
+              <ActionIcon
+                onClick={() => {
+                  setModalOpened(true)
+                  setSelectedHighlight(highlight)
+                }}
+              >
+                <IconNotes />
+              </ActionIcon>
+            </Group>
+          </Group>
+          <Spoiler maxHeight={120} showLabel={'Vis mer'} hideLabel="Hide">
+            <Text size={'sm'} color={'dimmed'}>
+              {highlight.description}
+            </Text>
+          </Spoiler>
+          <Modal
+            size={'lg'}
+            opened={modalOpened}
+            onClose={() => setModalOpened(false)}
+            overlayColor={theme.colors.gray[2]}
+            overlayOpacity={0.55}
+            overlayBlur={3}
```

Comment:
Overlay props, especially blur usually leads to lag on an application like this. Should get rid of it

Reply:
removed, although the screen gets completely black for me when removing overlayOpacity, so maybe that needs to be looked at from another device.